packages/components/organisms/f-header/src/tenants/en-NZ.js

```diff
@@ -49,5 +49,8 @@ export default {
     offers: {
         text: 'For you',
         url: '/offers'
-    }
+    },
+    selectYourCountryText: 'Select your country',
+    currentCountryKey: 'nz',
+    changeCurrentCountry: 'You are on the NZ website, click here to change'
```

Comment:
Is NZ approved copy here?
```suggestion
    changeCurrentCountry: 'You are on the New Zealandic website, click here to change'
```

Reply:
The same, we were provided with these versions for these tenants